modules/react/tokens/README.md

```diff
@@ -188,8 +188,8 @@ Variants only come their augmenting styles and a base type object is required.
 
 | Hierarchy Levels |
 | ---------------- |
-| `dataViz1`       |
-| `dataViz2`       |
+| `body1`       |
```

Comment:
Correct. It will look like this:

| Hierarchy Levels | Description                                           |
| ---------------- | ----------------------------------------------------- |
| `display`        | Intended for large heroes and images                  |
| `title`          | Intended for large page titles                        |
| `heading`        | Intended for headings and large text                  |
| `body`           | Intended for standard body text                       |
| `subtext`        | Intended for small subtext content or in tight spaces |

| Properties    | Description                                                      |
| ------------- | ---------------------------------------------------------------- |
| `fonts`       | Contains font-family tokens: `normal` and `monospace`            |
| `fontSizes`   | Contains font-size tokens: keys are in `px`, values are in `rem` |
| `fontWeights` | Contains font-weight tokens: `regular`, `medium`, and `bold`     |

| Variants  | Description                                       |
| --------- | ------------------------------------------------- |
| `error`   | Used for making errors more visible               |
| `hint`    | Used for help text and secondary content          |
| `inverse` | Used for any text on a dark or colored background |